aten/src/ATen/quantized/Quantizer.h

```diff
@@ -18,6 +18,23 @@
 
 namespace at {
 
+/**
+ * UnknownQuantizer is a placeholder quantizer for functions that implement
+ * quantization in a two step process.  First a tensor is allocated but with
+ * unknown quantizer, and then the quantization kernel decides what the final
+ * quantizer will be.
+ */
+struct TORCH_API UnknownQuantizer : public Quantizer {
+  explicit UnknownQuantizer(ScalarType scalar_type)
```

Comment:
is scalar_type going to be used in the quantization kernels? can we document this?

Reply:
I made UnknownQuantizer accept scalar type because the Quantizer superclass requires it. It shouldn't be used by anything. I'm actually a little surprised you store these on the Quantizer because it seems like you should be able to get them from the tensor.

Reply:
I see, yeah we need `scalar_type` because quantizer (as the name implies) is created before a quantized tensor, and it is able to quantize a floating point tensor to a tensor with a particular scalar type